src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+  color: $main-text-color;
```

Comment:
Ensure that the variable `$main-text-color` is defined in your SCSS files. If it's not defined, this will cause a compilation error.